autoware_launch/package.xml

```diff
@@ -6,16 +6,21 @@
 
   <maintainer email="yukihiro.saito@tier4.jp">Yukihiro Saito</maintainer>
   <maintainer email="isamu.takagi@tier4.jp">Takagi, Isamu</maintainer>
+  <maintainer email="akihiro.sakurai@tier4.jp">Akihiro Sakurai</maintainer>
   <license>Apache License 2.0</license>
 
   <buildtool_depend>ament_cmake_auto</buildtool_depend>
 
   <exec_depend>ad_api_adaptors</exec_depend>
+  <exec_depend>component_state_monitor</exec_depend>
+  <exec_depend>dummy_diag_publisher</exec_depend>
+  <exec_depend>emergency_handler</exec_depend>
```

Comment:
Do we need these dependency to individual packages? I think it is sufficient to just write `tier4_system_launch`.

Reply:
You're right. I removed those unnecessary dependencies.